mm/src/code/z_play.c

```diff
@@ -87,7 +87,7 @@ void Play_DrawMotionBlur(PlayState* this) {
         sMotionBlurStatus = MOTION_BLUR_OFF;
     }
 
-    MotionBlur_Override(&sMotionBlurStatus, &alpha);
```

Comment:
This needs to be reverted in your next commit.

Reply:
Reverted this and made some changes to get it synced up with the Modern Menu and the MenuBar.